galley/pkg/runtime/state.go

```diff
@@ -44,7 +44,7 @@ type State struct {
 
 	distribute  bool
 	strategy    *publish.Strategy
-	distributor publish.Distributor
+	distributor Distributor
```

Comment:
Can we make it `distributor` as well so it won't leak out of the package?

Reply:
It is meant to be passed in from the outside.